packages/office-ui-fabric-react/src/components/Toggle/docs/new/ToggleOverview.md

```diff
@@ -0,0 +1,5 @@
+A toggle represents a physical switch that allows someone to choose between two mutually exclusive options (e.g., on/off, show/hide), Choosing an option should produce an immediate result.
```

Comment:
```suggestion
A toggle represents a physical switch that allows someone to choose between two mutually exclusive options (e.g., on/off, show/hide). Choosing an option should produce an immediate result.
```

Reply:
I think it might be possible to suggest not only `two mutually exclusive options` but even `two opposite options` here.